suse-module-tools.spec

```diff
@@ -79,17 +67,14 @@ Recommends:     kmod
 # This release requires the dracut module 90nvdimm
 Conflicts:      dracut < 49.1
 # TW: conflict with pre-usrmerge
-%if 0%{?suse_version} >= 1550
-Conflicts:      filesystem < 15.5-40.2
-%endif
+Conflicts:      filesystem < 16
```

Comment:
Where did you get this from?

Reply:
The version in Leap 15.5 is 15.0, in TW is 84, ALP is identical.

Rather I would ask where that odd number is from, and the answer is likely some confusion during early usrmerge implementation.